IDL/relative_path_include.idl

```diff
@@ -0,0 +1,6 @@
+#include "../IDL/helpers/basic_inner_types.idl"

```

Comment:
Should not it be?

```suggestion
#include "../helpers/basic_inner_types.idl"
```

Reply:
All testing idl files are in the same folder. In order to test `..` it goes up a level just to go back to IDL.

Reply:
Sorry. I misread the path. It seems to be correct.